user-guide/Standard_Apps/SRM/srm_getting_started/config_service_orchestration/Service_Orch_configuring_resource_and_pools.md

```diff
@@ -0,0 +1,237 @@
+
+### Setting up additional pre-filtering
+
+To further limit the list of candidate resources, you can configure extra pre-filtering rules. This will generate an extra page in the Booking Wizard with controls that can be used to apply the extra filtering. <!-- RN 24172 -->
+
+To configure a pre-filtering rule:
+
+1. In the Services module, go to the *definitions* tab.
+
+1. Select the service definition and the node the filtering should apply to.
+
+1. In the *properties* pane below the diagram pane, add the property *Resource Assignment*, and set its value to a JSON string in the same format as the example below:
+
+   ```json
+   {
+      "Condition": "<A>",
+      "Value": [{
+            "Label": "A",
+            "Type": "Operation",
+            "Value": {
+               "FirstOperand": {
+                  "Link": "[var:ENC-Vendor]"
+               },
+               "Operator": "=",
+               "SecondOperand": {
+                  "Link": "RESOURCE",
+                  "Capability": "ENC-NewVendor"
+               }
+            }
+         }
+      ]
+   }
+   ```
+
+   The example above will generate an "ENC-Vendor" control that will list possible "ENC-NewVendor" capability values defined on the resources that are a candidate for the node.
+
+> [!NOTE]
+>
+> - Multiple conditions, separated by an "AND" operator, are supported. For example: `"Condition": "<A> AND <B>"`. Each condition must be of type "Operation", and each operation must be an equal comparison between a capability on a resource and a variable representing a field that will be presented to the user.
+> - In case the capability is also present on the profile instance, the list of profile instances will be filtered as well. <!-- RN 24034 -->
+
+> [!TIP]
+> See also: [Resource Assignment](xref:SRM_properties_Booking_Manager#resource-assignment)
+
+## Resource sorting configuration
+
+When candidate resources are presented in the Booking Wizard, these can be sorted in different ways.
+
+- **Based on priority**: To have them sorted based on priority, in the Resources module, add a *Priority* property to each resource in a pool. The lower the value you specify for this property, the higher in the sort order it will be displayed.
+
+  > [!TIP]
+  > See also: [Priority](xref:SRM_properties_Booking_Manager#priority)
+
+- **Alphabetically**: If priority is not specified, resources can be sorted alphabetically, if the *Alphabetically* option is selected for the *Resources Ordering Rule* setting on the *Config* > *Wizard* tab of the Booking Manager. <!-- RN 29107 -->
+
+- **Capacity**: If priority is not specified, resources can be sorted based on capacity, if the [Resource Sorting Capacity](xref:SRM_properties_Booking_Manager#resource-sorting-capacity) property is configured in the service definition and the *Capacity* option is selected for the *Resources Ordering Rule* setting on the *Config* > *Wizard* tab of the Booking Manager.
```

Comment:
based on available capacity ( considering all other overlapping bookings making use of that same resource)